FlashCards.RalphHo/Logic.cs

```diff
@@ -0,0 +1,176 @@
+using Microsoft.Data.SqlClient;
+
+public static class Logic
+{
+    //Create Flash Card
+    public static FlashCardModel CreateFlashCard(string Name, string Definition, int Id = 0, int StackId = 0)
+    {
+        FlashCardModel FlashCard = new FlashCardModel();
+        FlashCard.Name = Name;
+        FlashCard.Definition = Definition;
+        FlashCard.Id = Id;
+        FlashCard.StackId = StackId;
+        
+        return  FlashCard;
+    }
+    public static FlashCardModel SaveFlashCard(FlashCardModel model)
+    {
+        DBController.InsertFlashCard(DBController.ConnectDB(),model.Name,model.Definition,model.StackId);
```

Comment:
🟠 **Misplaced Responsibilities**

💡 Why do you have to do `DBController.ConnectDB()` in your Logic.cs?

🔧 This should be part of the DBController logic. Also it should be wrapped in a `using` statement/expression. e.g.
```
using var connection = new SqlConnection(connectionString);
connection.Open();
// DO SQL Stuff.
// connection.Close() is handled by the IDisposable implementation set up by adding the using syntax.
```